src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/methodparampad/Example2.java

```diff
@@ -0,0 +1,25 @@
+/*
+MethodParamPad
+allowLineBreaks = true
+option = space
+tokens = METHOD_DEF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.methodparampad;
+
+class Example2 {
+  public Example2() { 
+    super(); 
+  }
+
+  public Example2 (int aParam) {
+    super ();
+  }
+
+  public void method() {} // violation ''(' is not preceded with whitespace'
+
+  public void methodWithVeryLongName
+    () {}
```

Comment:
It would be good to either keep comment, or do `methodWithVeryLongName` -> `methodDeclarationWithLineWrap` to make it more obvious that we do not place a violation on this method because of line wrap.

Reply:
Done. I kept the comment.